webui/packages/eidolon-client/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "scripts": {
     "build": "tsup index.ts",
+    "watch": "tsup --watch",
```

Comment:
I see you added a watch command to the monorepos but ideally it would be added to webui because right now these have to be triggered individually--unless that's your intention

Reply:
These guys are triggered by the root level `turbo run watch` command when I was running locally